submission/loglist_refresher_test.go

```diff
@@ -201,7 +201,7 @@ func TestNewLogListRefresherUpdate(t *testing.T) {
 			} else if gotErr && !tc.errRegexp.MatchString(err.Error()) {
 				t.Fatalf("llr.Refresh() = (_, %q), want err to match regexp %q", err, tc.errRegexp)
 			}
-			if llNil, wantNil := ll == nil, tc.wantLL == nil; llNil != wantNil {
+			if llNil, wantNil := ll == nil, tc.wantLl == nil; llNil != wantNil {
 				t.Fatalf("llr.Refresh() = (%v, _), expected value? %t", ll, !wantNil)
```

Comment:
tiny suggestion:  you could simplify a touch further if you change the logged message to `"expected nil?"`

Reply:
Done